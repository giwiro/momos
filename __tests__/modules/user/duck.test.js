// @flow
import reducer, {
  UserUseFirstTime,
  initialState,
} from '../../../src/modules/user/duck';

describe('Action creators', () => {
  it('should return an action to erase the first time flag', () =>
    expect(new UserUseFirstTime()).toMatchSnapshot());
});

describe('Reducer', () => {
  it('should return initialState', () =>
    expect(initialState).toMatchSnapshot());

  it('should return the state with error set', () =>
    expect(reducer(initialState, new UserUseFirstTime())).toMatchSnapshot());
});
