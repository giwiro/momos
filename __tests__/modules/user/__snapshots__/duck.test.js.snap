// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Action creators should return an action to erase the first time flag 1`] = `
Object {
  "type": "UserUseFirstTime",
}
`;

exports[`Reducer should return initialState 1`] = `
Object {
  "firstTime": true,
}
`;

exports[`Reducer should return the state with error set 1`] = `
Object {
  "firstTime": false,
}
`;
